Tests/SonarScanner.MSBuild.Test/SonarScanner.MSBuild.Test.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net5.0</TargetFrameworks>
+    <TargetFrameworks>net48</TargetFrameworks>
+    <LangVersion>8.0</LangVersion>
```

Comment:
Why is the LangVersion change needed?

Reply:
the build was failing locally because the default language version is `7.3` for `net48` instead of `8.0`.

I opened https://github.com/SonarSource/sonar-scanner-msbuild/issues/1167 to consolidate

![image](https://user-images.githubusercontent.com/38876598/151537643-042f6fea-cf09-4694-98b6-133e5674761f.png)